icu4c/source/common/unicode/rbbi.h

```diff
@@ -71,32 +71,38 @@ class U_COMMON_API RuleBasedBreakIterator /*U_FINAL*/ : public BreakIterator {
      * Not for general use; Public only for testing purposes.
      * @internal
      */
-    RBBIDataWrapper    *fData;
+    RBBIDataWrapper    *fData = nullptr;
+
 private:
+    /**
+      * The saved error code associated with this break iterator.
+      * This is the value to be returned by copyErrorTo().
+      */
+    UErrorCode      fErrorCode = U_ZERO_ERROR;
```

Comment:
Note for Reviewers: talking about copyErrorTo() already is jumping the gun a bit on an upcoming PR, which will introduce it as its primary point. But adding fErrorCode is needed now to allow the default constructor to delegate to the one taking a UErrorCode.